src/options.js

```diff
@@ -48,6 +48,8 @@ import schema from './options.json';
 export function getOptions(pluginOptions) {
   const options = {
     extensions: 'js',
+    emitError: true,
+    emitWarning: true,
```

Comment:
warnings should not fail build

Reply:
Exact, so failOn Warnings is not enabled by default

Reply:
`emitWarning` its diferent that `failOnWarning`

Reply:
Yep, I see, my mistake

Reply:
No problem, too many plugins to you review :grin: